client/app/components/QueryBasedParameterInput.jsx

```diff
@@ -42,10 +45,23 @@ export class QueryBasedParameterInput extends React.Component {
     }
   }
 
+  async _fetchOptions() {
+    try {
+      return await this.props.parameter.loadDropdownValues(this.props.isDirty);
+    } catch {
+      if (this.props.isDirty) {
+        notification.error('Cannot Fetch Dropdown Parameter Values', 'This query has other queries associated with it as dropdown parameters. ' +
+        'In order to edit this query, you must have access to the associated queries.', { duration: 10 });
```

Comment:
```js
notification.error(
  'Cannot Fetch Dropdown Parameter Values', 
  `
    This query has other queries associated with it as dropdown parameters. 
    In order to edit this query, you must have access to the associated queries.
  `, 
  { duration: 10 },
);
```

Reply:
Wouldn't that add a line break to the formatted text?

Reply:
It's html - browser will replace newline with whitespace, and then collapse all whitespaces.